cmd/ib-sriov-cni/main.go

```diff
@@ -230,6 +264,13 @@ func cmdDel(args *skel.CmdArgs) error {
 	}
 	defer netns.Close()
 
+	// Lock CNI operation to serialize the operation
+	lock, err := lockCNIExecution()
```

Comment:
It is already locked from line 113

Reply:
good catch, ill remove

Reply:
looking at this again, i originally thought that i mistakenly locked twice. but this is not the case.
this lock is for cmdDel and on line 113 its for cmdAdd